lib/http/utils/load-wallet.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 
 const keysDirectory = process.env.COINSENCE_WALLETS_DIR || path.join(__dirname, '../../../', 'keys');
 
-module.exports = function (accountId, password) {
+module.exports = async function (accountId, password) {
```

Comment:
why is this needed here?

Reply:
mmm to use `await` here https://github.com/Coinsence/coinsence-monorepo/pull/29/files#diff-acff7027fa464ebffb6774cfb33669cbR40 to wait for the returned promise...
what do u think ?

Reply:
I think that is not needed, this function returns a promise anyway. and you need an async on the function that is using await.